estate/models/estate_property.py

```diff
@@ -0,0 +1,20 @@
+from odoo import fields, models
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "estate property info"
+
+    name = fields.Char(required=True, translate=True)
+    description = fields.Text(translate=True)
+    postcode = fields.Char()
+    date_availability = fields.Date()
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(string='garden_orientation', selection=[('North', 'north'), ('east', 'East'), ('West', 'west'), ('South', 'south')])
```

Comment:
string is the name of the field that will be displayed to the user. Therefore it should be formatted as such, in title case

The selection list should be a list of tuples as such:
`[('database_value', "Display Value"), ...]`
usually we snake case the database value and title case the user-facing value. Here is an [example](https://github.com/odoo-dev/odoo/blob/19.0-peppol-rate-limit-ui-roan/addons/account/models/account_account_tag.py#L12)

```suggestion
    garden_orientation = fields.Selection(string='Garden Orientation', selection=[('north', 'North'), ('east', 'East'), ('west', 'West'), ('south', 'South')])
```